report/renderer/category-renderer.js

```diff
@@ -531,6 +530,7 @@ export class CategoryRenderer {
 
     // Sort audits into clumps.
     for (const auditRef of category.auditRefs) {
+      if (auditRef.group === 'hidden') continue;
```

Comment:
This was actually a bug. Hidden audits marked N/A would show up in the report because the N/A clump is rendered using `renderClump` instead of `renderUnexpandableClump`

Reply:
I guess it's not related, but tests only started failing once the artifacts were refreshed. I'll split it out...